scripts/ci/requirements.txt

```diff
@@ -14,7 +14,7 @@ flake8
 flask
 flask-admin
 flask-cache
-flask-login
+flask-login==0.2.11
```

Comment:
We should just move to 0.3 and pin that version here as well - AFAICT there's no reason we can't be on 0.3